Fix bug with config file defaults

brianhlin opened the pull request:
We missed this in #79

DrDaveD commented on rpm/osg-pki-tools.spec:
```diff
@@ -43,6 +43,9 @@ mv %{buildroot}/%{_prefix}/config/ca-issuer.conf %{buildroot}%{_sysconfdir}/osg/
 %changelog
+* Wed Sep 6 2023 Brian Lin <blin@cs.wisc.edu> - 3.6.1
+- Fix bug with default CA config file option (SOFTWARE-5668)
+
```
Suggest inserting also the updates of igtfservercert and igtfmultidomain done in #80

brianhlin replied:
Good call. Fixed

brianhlin commented:
@mwestphall this is now ready for your review I added some exception handling for issues reading the config file. I didn't add exceptions for missing config attributes because those are handled by the wide except here https://github.com/opensciencegrid/osg-pki-tools/blob/master/osgpkitools/incommon_request.py#L380-L382

I also moved the default org/dept configs to the config file instead of in the options as the latter is surprising now that we actually have a config file.
